tests/ha/clvm.pm

```diff
@@ -112,7 +112,6 @@ sub run {
         assert_script_run "sed -ie '/\\<volume_list\\>[[:blank:]][[:blank:]]*=/ a volume_list = [ $vol_list ]' $lvm_conf";
     }
 
-
```

Comment:
really strange change . unless travis insists I would drop this from PR

Reply:
Not sure that Travis insists on it, it's just that I found this not useful blank line.
I can remove it if you want, but it's not hurting anything :)

Reply:
My point here that after merging this PR if someone will take a look into history of clvm.pm file he/she will find commit which states "SLE HA fixes" but in a fact removing blank line not related to "SLE HA fixes" anyhow. This is why I think it is better collect such changes into separate commit which clearly state "Removing blank lines" or doing so when you also have some meaningful changes in this file. Otherwise you creating "noise" for someone who trying to use git history

Reply:
Ok, I created a new commit only for this small change.